### DISCUSSION TITLE
What invertype type to choose for a inverter type is MOD 7000TL3-XH

### DISCUSSION THREAD

**Discussion by SpoturDeal** [johanmeijer/grott#615] [Q&A]
I have a Growatt MOD 7000TL3-XH now I have set in the grott.ini the inverter type as default. Is this the best option?

**Comment by FlexxFR** [comment]
I have a MOD 8000TL3-XH and learned that the default option does work for basic values but with the latest version there is support for intertype = mod.
This also include battery values.

**Comment by SpoturDeal** [comment]
Thank you, it works. However the wiki needs an update.
